### ISSUE TITLE
test results constantly lost

### ISSUE THREAD

**Issue by lolplusultra** [kondratyev-nv/vscode-python-test-adapter#313]
I am remote connecting to a raspberry pi which ist running the tests.
When I run a test the result is showing up in the tree correctly.
But after some days the old results start to disappear and only newer runs are kept.
Is there some kind of queue in place?

**Comment by kondratyev-nv** [comment]
No, there's no queue. Do you continuously run them? Do you change code between runs?
